static/js/util/profile_edit.js

```diff
@@ -281,44 +281,10 @@ export function boundCountrySelectField(stateKeySet, countryKeySet, label) {
  *
  * @param keySet {String[]} Path to look up and set a field
  * @param label {String} Label for the field
+ * @param omitDate {bool} If false, there is a date textbox in addition to month and year
```

Comment:
shouldn't this be `omitDay`? when this is true, we just intend to omit the day input, right?